internal/covid/covidclient/client.go

```diff
@@ -0,0 +1,61 @@
+package covidclient
```

Comment:
ทำเป็น go generate ตัว mock 

```suggestion
package covidclient
```
```suggestion
//go:generate mockgen -source=./client.go -destination=./mock/mock_client.go -package=mock

package covidclient
```